doc/quick.md

```diff
@@ -1,21 +1,16 @@
 # Quickstart
 
-This has steps have been tested on:
+These steps have been tested on:
 
 * kubeadm (vanilla k8s cluster),
-* Google Kubernetes Engine (GKE) cluster using Ubuntu image
-* Rancher
-* RKE + Fedora CoreOS
-    * You will need to add following directory mapping to `services_kubelet->extra_binds` in your `cluster.yml`:
-     `/opt/rke/var/lib/kubelet/plugins:/var/lib/kubelet/plugins`. Otherwise the CSI socket paths won't match and the CSI
-     driver registration process will fail.
 
 ### Requirements
 
  * Only works on Intel Core i7 processor architecture (march=corei7) and higher
  * MayaStor Daemonset (MDS) requires privileged mode
- * MDS requires 2MB hugepages
+ * MDS requires 2MB huge pages
```

Comment:
We require a huge page *size* of 2MB  and require at least 512 of those. We will bump that in the future but technically, 1GB is enough right now.